frontend/src/app/core/global_search/input/global-search-input.component.sass

```diff
@@ -108,6 +108,10 @@ $search-input-height: 30px
       max-height: 80vh
       height: auto
 
+    .ng-dropdown-panel
+      width: 520px !important
+      transform: translateX(-10%) !important
```

Comment:
This looks super weird on "normal" screen sizes to me

<img width="599" alt="Bildschirmfoto 2025-05-09 um 10 54 02" src="https://github.com/user-attachments/assets/124fdff8-0bb6-4c59-9a74-9df10c158ffc" />

Maybe we need to switch to the mobile behaviour earlier or make it a max-width instead.